addon/components/cfb-form-editor/question/default.js

```diff
@@ -0,0 +1,83 @@
+import { getOwner } from "@ember/application";
+import { action, computed } from "@ember/object";
+import RenderComponent from "ember-validated-form/components/validated-input/-themes/uikit/render";
+
+import { TYPE_MAP } from "ember-caluma/lib/field";
+
+export default class CfbFormEditorQuestionDefault extends RenderComponent {
+  @computed("model.{id,__typename}")
+  get question() {
+    const raw = {
+      ...this.model.get("data"),
+      ...this.model.get("change"),
+    };
```

Comment:
Not 100% sure but I think I used `changeset.unwrap()` for this when I had the same usecase

Reply:
I'm probably not completely awake yet :smile: 

I could swear that I've seen `.unwrap` somewhere lately. But I can't find it in the ember-changeset read-me :thinking: 

The idea of merging the data with the changes comes from [cfb-form-editor/question/options](https://github.com/projectcaluma/ember-caluma/blob/master/addon/components/cfb-form-editor/question/options.js#L64-L68).

Reply:
Apparently it's not public API, however I think the proper way would be to use `this.model.execute()` ([docs](https://github.com/validated-changeset/validated-changeset#execute)) and then use `this.model.data`

Reply:
Would this be save? I'm hesitant because the model is for the whole question and not just the default answer from this component. I really don't know enough about the form-builder to judge if this could have adverse effects on other parts of the form or not.

Reply:
`save()` will call the underlying objects `save()` method after running `execute()`, we only want `execute()`

Reply:
My bad! I meant execute. As this changes the `.data` I figure this might have some consequences for `cfb-form-editor/question.js`. Or maybe the validated form that's used there. But I don't know. That's why I like the idea of not mutating `this.model`. But if you tell me that this is not a problem, I will use `raw = { ...this.model.data }` (with the execute beforehand).

Reply:
Hm I see your point.. I don't know what would happen, maybe try it? Otherwise keep it that way or use `unwrap()` I'll leave that up to you.